src/Documents.jl

```diff
@@ -253,14 +253,9 @@ function Document(;
         sitename :: AbstractString   = "",
         authors  :: AbstractString   = "",
         analytics :: AbstractString  = "",
-        version :: AbstractString    = "",
-        html_prettyurls  :: Bool     = true,
-        html_disable_git :: Bool     = false,
-        html_edit_branch :: Union{String, Nothing} = "master",
-        html_canonical   :: Union{String, Nothing} = nothing,
```

Comment:
We should keep these around with deprecation warnings though. I'd say:

* If `HTML` _and_ any of these are passed, throw an error.
* If only keywords are passed, construct an `HTML` object out of them, but also print a depwarn.